src/js/common/globalOrLiteral.js

```diff
@@ -0,0 +1,27 @@
+/* global require */
+(function (fluid) {
+    "use strict";
+    fluid = fluid || require("fluid");
+    var gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.express.user");
+
+    /**
+     *
+     * Evaluate a single entry and retrieve the associated global variable if possible.
+     *
+     * @param {String|Object} toEvaluate - A single message bundle represented as a raw object or as a global variable name.
+     * @return {Object} - A message bundle object.
```

Comment:
Note that it might not be of type {Object} if some value other than undefined or an Object is stored in the global name.

Reply:
It's gone.